src/Compilers/CSharp/Test/Syntax/Parsing/SuppressNullableWarningExpressionParsingTests.cs

```diff
@@ -252,5 +252,363 @@ public void ConditionalAccess()
+                        {
+                            N(SyntaxKind.MemberBindingExpression);
+                            {
+                                N(SyntaxKind.DotToken);
+                                N(SyntaxKind.IdentifierName);
+                                {
+                                    N(SyntaxKind.IdentifierToken, "y");
+                                }
+                            }
+                            N(SyntaxKind.ExclamationToken);
+                        }
+                        N(SyntaxKind.ExclamationToken);
+                    }
+                    N(SyntaxKind.DotToken);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "z");
+                    }
+                }
+            }
+            EOF();
+        }
+
+        [Fact]
+        public void ConditionalAccess_Repeated_02()
+        {
+            UsingNode("x?.y!!!!.z");
+            N(SyntaxKind.ConditionalAccessExpression);
+            {
+                N(SyntaxKind.IdentifierName);
+                {
+                    N(SyntaxKind.IdentifierToken, "x");
+                }
+                N(SyntaxKind.QuestionToken);
+                N(SyntaxKind.SimpleMemberAccessExpression);
+                {
+                    N(SyntaxKind.SuppressNullableWarningExpression);
+                    {
+                        N(SyntaxKind.SuppressNullableWarningExpression);
+                        {
+                            N(SyntaxKind.SuppressNullableWarningExpression);
+                            {
+                                N(SyntaxKind.SuppressNullableWarningExpression);
+                                {
+                                    N(SyntaxKind.MemberBindingExpression);
+                                    {
+                                        N(SyntaxKind.DotToken);
+                                        N(SyntaxKind.IdentifierName);
+                                        {
+                                            N(SyntaxKind.IdentifierToken, "y");
+                                        }
+                                    }
+                                    N(SyntaxKind.ExclamationToken);
+                                }
+                                N(SyntaxKind.ExclamationToken);
+                            }
+                            N(SyntaxKind.ExclamationToken);
+                        }
+                        N(SyntaxKind.ExclamationToken);
+                    }
+                    N(SyntaxKind.DotToken);
+                    N(SyntaxKind.IdentifierName);
+                    {
+                        N(SyntaxKind.IdentifierToken, "z");
+                    }
+                }
+            }
+            EOF();
+        }
+
+        [Fact]
+        public void ConditionalAccess_Repeated_03()
+        {
+            UsingNode("x?.y.z!!");
```

Comment:
aside... allowing `!` at the end of `... ?. ...` is *SUPER* weird right?  like no good codebase should ever have that.  the point of using `?.` is to say "i need to handle null in this expr, and i want to shortcircuit to `null` in that case".  But then you're explicitly overriding that at the end with a `!` to say *"i will never get null"* ??

if you can't ever get null, then why use `?.` in teh first place.  and, if you're using `?.`, then you are saying "i can get null and will have to deal with it"

Note: this is not about your PR, but it feels like a def place for an analyzer to ask what hte heck you're doing :)